spec/order_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'order'
+
+describe Order do
+
+  let(:menu) do
+
+    double :menu, view: {pepperoni:  10.0,
+                         margharita:  8.0}
+
+  end
+
+  subject {Order.new(menu)}
+
+  it {expect(subject).to respond_to(:add).with(2).argument}
+
+  it 'should let customers see their order' do
+    expect(subject.view_order).to eq ({})
+  end
+
+  it 'should let customers see their order total' do
+    expect(subject.view_total).to eq 0
+  end
+
+  context 'add should' do
+    it 'add a single pizza to the selection' do
+      subject.add(:pepperoni, 1)
```

Comment:
Could the item encapsulate its own value? That way you wouldn't have to use `Menu` like a dictionary